src/common/session/session-entity.js

```diff
@@ -25,9 +25,10 @@ const model = {
   inactiveAt: 0,
   expiresAt: 0,
   updatedAt: Date.now(),
-  sessionReplay: MODE.OFF,
+  sessionReplayMode: MODE.OFF,
   sessionReplaySentFirstChunk: false,
   sessionTraceMode: MODE.OFF,
+  traceHarvestStarted: false,
```

Comment:
Is this necessary since you will be able to tell that a session `isNew` per page?  only report firstHarvest flag if isNew?

Reply:
Falls into the same idea that the SR has with the reason why SentFirstChunk was added -- the real first trace payload can come from any tab theoretically.

This isn't an issue with RUM (flag) however, bc that is unconditionally always sent at load of a new tab, basing off `isNew` makes sense there.

Reply:
Just want to double check here -- Is there a race condition though were both tabs could read from session state as `traceHarvestStarted: false` before it has updated, theoretically sending 2 payloads with 

`!this.agentRuntime.session.state.traceHarvestStarted`

Reply:
Yeah, so the new `SESSION_EVENTS.UPDATE` just needs to be listened on, will add into stn

Reply:
Wait, the session-entity listener to storage updates already manages this. That will update the session.state object.